vagrant/config/versions.yaml

```diff
@@ -101,6 +101,20 @@ installers:
       - 'ubuntu2004'
       - 'ubuntu2204'
 
+  - foreman: '3.12'
+    katello: '4.14'
+    candlepin: '4.4'
+    pulpcore: '3.49'
+    puppet: 8
+    boxes:
+      - 'almalinux8'
+      - 'almalinux9'
+      - 'centos8-stream'
```

Comment:
We should really drop it everywhere, but perhaps just fix 3.12 at least?
```suggestion
```

Reply:
removed